src/main/java/com/airsquared/blobsaver/Background.java

```diff
@@ -334,14 +356,11 @@ private static void saveBackgroundBlobs(int preset) {
         }
     }
 
-    private static void showStage() {
-        primaryStage.show();
-        primaryStage.toFront();
-        primaryStage.requestFocus();
-    }
-
     static void stopBackground(boolean showAlert) {
         inBackground = false;
+        /*if (PlatformUtil.isMac()) {
```

Comment:
Could you please remove this unused comment?

Reply:
Done.